src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -417,7 +426,12 @@
                 this.donationMatchingOptions = info.donationMatchingOptions;
                 this.donationMatchingRule = info.donationMatchingRule;
                 this.donationDateRange = info.donationDateRange;
+                this.labels = info.labels;
                 this.postProcessClass = info.postProcessClass;
+                this.processUsingScheduledJob = info.processUsingScheduledJob;
+                this.noMatchSelected = (info.donationMatchingBehavior === this.labels.DoNotMatch);
+                this.noMatchOnDate = (info.donationMatchingRule.indexOf("donation_date__c") < 0);
```

Comment:
In my namespaced org, donation date is always disabled because the value is equal to npsp__donation_date__c instead of donation_date__c. This might need some more complex logic to work here.

Reply:
Yup. Good call.